src/babel/transformation/transformers/es7/function-bind.js

```diff
@@ -0,0 +1,41 @@
+// https://github.com/zenparsing/es-function-bind
+
+import * as t from "../../../types";
+
+export var metadata = {
+  optional: true,
+  stage: 0
+};
+
+function getTempId(scope) {
```

Comment:
Yeah - and sequence of operations is always like:
1. Store context to `_context`.
2. Get `.bind` / `.call` property.
3. Call it with `_context` as first argument.

So, according to semantics, nothing can happen between storing and using `_context`.